desk/app/channels.hoon

```diff
@@ -575,6 +575,7 @@
     ==
   ++  ca-abed
     |=  n=nest:c
+    ?.  (~(has by v-channels) n)  ca-core
```

Comment:
I don't think this is actually good and could end up doing weird things. I tried moving up to line 541 and doing some no-oping there but I'm not sure that's right either.

Reply:
Yeah this changes the semantics of `+ca-abed` enough to make a lot of the code here "clearly wrong".

Was gonna say that, what we want instead, is a `%u` scry for channel existence, but we already have it! Whoever is doing a scry that fails on the `got:by` here should first be doing the `%u` version of the scry.